.github/workflows/reusable-build-test-config.yml

```
@@ -166,6 +174,10 @@ jobs:
           BUILD_TYPE: ${{ inputs.build_type }}
           PARALLELISM: ${{ runner.os == 'Windows' && '1' || steps.nproc.outputs.nproc }}
         run: |
+          export ASAN_OPTIONS="detect_container_overflow=0 suppressions=$GITHUB_WORKSPACE/sanitizers/suppressions/asan.supp"
+          export TSAN_OPTIONS="second_deadlock_stack=1 halt_on_error=0 suppressions=$GITHUB_WORKSPACE/sanitizers/suppressions/tsan.supp"
+          export UBSAN_OPTIONS="suppressions=$GITHUB_WORKSPACE/sanitizers/suppressions/ubsan.supp"
+          export LSAN_OPTIONS="suppressions=$GITHUB_WORKSPACE/sanitizers/suppressions/lsan.supp"
```

Nit: Using `${xxx}` over `$x` is preferred.

```suggestion
          export ASAN_OPTIONS="detect_container_overflow=0 suppressions=${GITHUB_WORKSPACE}/sanitizers/suppressions/asan.supp"
          export TSAN_OPTIONS="second_deadlock_stack=1 halt_on_error=0 suppressions=${GITHUB_WORKSPACE}/sanitizers/suppressions/tsan.supp"
          export UBSAN_OPTIONS="suppressions=${GITHUB_WORKSPACE}/sanitizers/suppressions/ubsan.supp"
          export LSAN_OPTIONS="suppressions=${GITHUB_WORKSPACE}/sanitizers/suppressions/lsan.supp"
```

---

Done